src/MessagePack.Annotations/MessagePack.Annotations.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.6;netstandard2.0;net45;net47</TargetFrameworks>
```

Comment:
We don't need so many targets here. I suspect `netstandard1.0` is adequate by itself.

Reply:
Sure, will do today